src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private String color;
```

Comment:
You do not need to create field color, as Figure class already has this field
Please, remove it
Same for all others figures